monty/data.py

```diff
@@ -18,3 +18,18 @@ def download_if_not_present():
 
     logger.info('done downloading dataset')
     return local_file_path
+
+
+def create_dataset(path_to_raw_dataset, num_epochs=10, batch_size=128, shuffle_buffer_size=1000, row_count=2700):
+    filenames = [path_to_raw_dataset]
+    record_defaults = [tf.int64] * row_count
+    dataset = tf.contrib.data.CsvDataset(filenames, record_defaults, header=True,
+                                         select_cols=[x for x in range(1, row_count + 1)])
```

Comment:
no list comprehension needed:
[x for x in range(1, row_count + 1)] = range(1, row_count + 1)

Reply:
nope, range returns a range object not a list and tf wants a list :(

Reply:
list(range(1, row_count + 1)) will do though